.appveyor.yml

```diff
@@ -4,6 +4,9 @@ version: '{branch}-{build}'
 # Do not build on tags (GitHub and BitBucket)
 skip_tags: true
 
+# Limit the number of last commits when cloning the repo/branch
+clone_depth: 1
```

Comment:
This makes clone go from 2+mins to 10sec or less in appveyor. More info here https://www.appveyor.com/docs/how-to/repository-shallow-clone/

Reply:
cc @sfsholden first step towards faster appveyor runs :)

Reply:
Only thing I noticed on this page was a warning about setting the number too low. I don't know if this would cause an issue for us or not.

"Be aware that if you do a lot of commits producing queued builds and depth number is too small git checkout operation following git clone can fail because requested commit is not present in a cloned repository."

Reply:
Yeah, I haven't seen us commit fast enough to run into this but if we start running into issues we can always modify this.